### PR TITLE
Pub 2089 - Align JOH functionality

### PR DESCRIPTION
**Pull request by NatashaAlker**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-2089

### Change description ###

Update validation schemas of impacted lists to align JOH properties

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/schemas/cop_daily_cause_list.json [hunk 1]
```diff
@@ -174,25 +174,6 @@
                 "$id": "#root/locationDetails/region/regionalJOH/items",
                 "type": "object",
                 "properties": {
-                  "johTitle": {
```
Comment (KianKwa) [inline src/main/resources/schemas/cop_daily_cause_list.json:177]: The COP list does not currently have the isPresiding flag in the schema. Just checking this is expected?

File: src/main/resources/schemas/sscs_daily_list.json [hunk 2]
```diff
@@ -31,34 +31,6 @@
       "description": "Can be an array of objects as such will be a collective and use plurals for object arrays (clerks and scribes etc included)",
       "type": "object",
       "properties": {
-        "johTitle": {
```
Comment (KianKwa) [inline src/main/resources/schemas/sscs_daily_list.json:34]: The SSCS list too does not currently have the isPresiding flag in the schema. Just checking this is expected?

File: src/test/resources/mocks/cop-daily-cause-list/copDailyCauseList.json [hunk 3]
```diff
@@ -65,7 +63,7 @@
                 ],
                 "judiciary": [
                   {
-                    "johKnownAs": "Crown Judge",
+                    "johKnownAs": "Judge TestName Presiding",
                     "isPresiding": true
```
Comment (KianKwa) [inline src/test/resources/mocks/cop-daily-cause-list/copDailyCauseList.json:67]: COP list schema does not have isPresiding flag currently. Need to double check this. If so all COP test files should not have this field.

File: src/main/resources/schemas/cop_daily_cause_list.json (hunk 1 above)
Comment (NatashaAlker) [reply src/main/resources/schemas/cop_daily_cause_list.json:177]: Have checked with Chris and the COP list regionalJOH doesn't need the isPresiding flag adding

File: src/main/resources/schemas/sscs_daily_list.json (hunk 2 above)
Comment (NatashaAlker) [reply src/main/resources/schemas/sscs_daily_list.json:34]: Checked with Chris and this one did need the isPresiding flag adding, I've made those changes now

**Review by KianKwa — APPROVED** [review]

**Pull request merged**